alerts/google-gke/uptime-checks-workload.v1.json

```diff
@@ -0,0 +1,37 @@
+{
+  "displayName": "${CLUSTER_NAME}/${WORKLOAD_NAME} GKE Load Balancer Check uptime failure"
```

Comment:
I think this is fine for now but note we can technically support uptime checks for ingress via URL, so we may want to consider passing in the "Load Balancer" part of the display name